src/scheduler/gc_work.rs

```diff
@@ -10,6 +10,12 @@ use crate::*;
 use std::marker::PhantomData;
 use std::ops::{Deref, DerefMut};
 
+/// Buffer size for [`ProcessEdgesWork`] work packets. This constant is exposed to users so that
+/// they can use this value for places in their binding that interface with the work packet system,
+/// specifically the transitive closure via `ProcessEdgesWork` work packets such as roots gathering
+/// code or weak reference processing.
```

Comment:
I think the key statement is missing: we expect users to split their workload by this buffer size for process-edges-related work, so each work packet should include at most this many items in the buffer.

Reply:
Done.